src/github.com/getlantern/lantern-ui/app/angular-feed/scripts/services.js

```diff
@@ -37,7 +37,8 @@ angular.module('feeds-services', []).factory('feedService', ['$q', '$http', '$sc
           deferred.reject(new Error("invalid data format"));
           return
         }
-        feedCache.set(feedURL, data);
+        // Always get fresh copy, in hope to be more reactive to network errors
+        // feedCache.set(feedURL, data);
```

Comment:
Sounds like maybe we should keep it then? Very odd it could sometimes take so long. Is your proxy generally slow at times when you've seen it take a long time to load the feed?